src/Timeline.js

```diff
@@ -187,6 +213,22 @@ export default class Timeline extends Axis {
       .call(attrize, this._handleConfig)
       .attr("height", timelineHeight + this._handleSize);
 
+    if (this._buttonBehaviorCurrent === "buttons") {
+
+      const yTransform = this._height / 2 - this._buttonHeight / 2;
+
+      this._brushGroup.selectAll(".selection")
+      .call(attrize, this._selectionConfig)
+      .attr("height", timelineHeight)
+      .attr("y", yTransform);
+
+      this._brushGroup.selectAll(".handle")
+      .call(attrize, this._handleConfig)
+      .attr("height", this._buttonHeight)
+      .attr("y", yTransform);
```

Comment:
same comment for these settings, you're duplicating the `attrize` call, move it inline with the above statements